WebPortal/Pages/Admin/PowerBIAdmin.razor

```diff
@@ -431,8 +449,21 @@
     private PowerBiAdminWorkspaceTreeItem WorkspaceBeingEdited { get; set; }
     private PowerBiAdminDatasetTreeItem DatasetBeingEdited { get; set; }
     private PowerBiAdminReportTreeItem ReportBeingEdited { get; set; }
+    private string _groupWithId;
 
-    private bool IncludeCurrentItemInCatalog { get; set; } = true;
```

Comment:
This needs to stay separate from ReportBeingEdited, since it's also used to add and remove workspaces and datasets from the catalog. I think you could accomplish the data side of this by setting IsCatalog on the report when saving it.